bleak/backends/bluezdbus/client.py

```diff
@@ -513,6 +513,9 @@ async def read_gatt_char(
             (bytearray) The read data.
 
         """
+        if self._bus is None:
+            raise BleakError("D-Bus object is None")
```

Comment:
How about saying that the client is not connected instead? Isn't that what triggers this problem?

Reply:
Good point.  Let me look into this one more and clean up the error message.